corona-warn-app/cwa-app-android
# Fix:  Reminder-Notification (EXPOSUREAPP-12685)

Use `testIdentifier` in the reminder notification

### Testing
- Scan positive PCR/RAT test and get results
- don't warn others
- wait until you see reminder notification with text "Bitte warnen Sie andere und teilen Sie Ihr Testergebnis"
- tap on the notification and you should be able to complete the submission flow

**Testing tip:** notification is scheduled for 2h after the test result has been received but you can decrease waiting time by playing with values of the `POSITIVE_RESULT_NOTIFICATION_INITIAL_OFFSET` and `POSITIVE_RESULT_NOTIFICATION_INTERVAL` in your local branch.

### Jira Ticket
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-12685
